prac_06/used_cars.py

```diff
@@ -9,10 +9,16 @@
 
 def main():
     """Demo test code to show how to use car class."""
-    my_car = Car(180)
+    my_car = Car("My Car", 180)
     my_car.drive(30)
     print(f"Car has fuel: {my_car.fuel}")
     print(my_car)
 
+    limo = Car("Limo", 100)
+    limo.add_fuel(20)
+    print(limo.fuel)
```

Comment:
Could you format this print statement similar to the previous "Car has fuel: {my_car.fuel}"? Just a small suggestion!